src/styles/blocks/header.scss

```diff
@@ -0,0 +1,13 @@
+.header {
+  @extend %flex-between-center;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+
+  &__nav-bar {
+    margin-right: 50px;
+  }
+
+  &__logo {
+    margin: 10px 0 10px 50px;
```

Comment:
Strange solution
1 - you could make header flexible and align items in it (what if we decide to make links higher?)
2 - it's more logical to set paddings for parent container instead of margins for both of it's children